services/web/server/tests/integration/01/test_garbage_collection.py

```diff
@@ -635,7 +635,7 @@ async def test_t4_project_shared_with_group_transferred_to_user_in_group_on_owne
     await assert_projects_count(aiopg_engine, 1)
     await assert_user_is_owner_of_project(aiopg_engine, u1, project)
 
-    await asyncio.sleep(WAIT_FOR_COMPLETE_GC_CYCLE)
+    await asyncio.sleep(2 * WAIT_FOR_COMPLETE_GC_CYCLE)
```

Comment:
flake fix?

Reply:
This is a very interesting one. But difficult to explain. I discussed it with @matusdrobuliak66 . Will check offline again with you. 

In summary: we MUST AVOID `sleep` in tests.